### PR TITLE
Update orders-service OpenAPI spec in getting-started guide

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:

As in title: Update orders-service OpenAPI spec in getting-started guide

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/4095

### CODE REVIEW

File: docs/getting-started/assets/orders-service-openapi.yaml [hunk 1]
```diff
@@ -124,14 +124,15 @@ components:
         consignmentCode:
           type: string
           example: "123456789"
-          description: ID of the order consignment code
+          description: ID of the consignment code of order
```
Comment (kazydek) [inline docs/getting-started/assets/orders-service-openapi.yaml:127]:
```suggestion
          description: ID of the order consignment code
```

File: docs/getting-started/assets/orders-service-openapi.yaml [hunk 2]
```diff
@@ -124,14 +124,15 @@ components:
...
         consignmentStatus:
           type: string
           example: "PICKUP_COMPLETE"
-          description: Order consignment status
+          description: Consignment status of order
```
Comment (kazydek) [inline docs/getting-started/assets/orders-service-openapi.yaml:131]:
```suggestion
          description: Order consignment status
```

**Review by kazydek — COMMENTED** [review]
two tiny bits

**Review by kazydek — APPROVED** [review]

**Review by aerfio — APPROVED** [review]

**Pull request merged**